src/Database/TigerBeetle/Internal/FFI/Client.hsc

```diff
@@ -117,10 +118,10 @@ instance Enum TBOperation where
     toEnum (#const TB_OPERATION_GET_EVENTS)            = GetEvents
     toEnum unmatched = error $ "TBOperation.toEnum: Cannot match " ++ show unmatched
 
-marshallTBOperation :: TBOperation -> Word8 
+marshallTBOperation :: TBOperation -> Word8
```

Comment:
My editor is configured to cleanup whitespace and I figured I'd leave these changes in. I recommend turning off whitespace differences when reviewing.